test/commands/account/get.test.js

```diff
@@ -117,5 +118,33 @@ describe('account:get command', () => {
 				]);
 				return expect(printMethodStub).to.be.calledWithExactly(queryResult);
 			});
+
+		setupTest()
+			.stub(query, 'default', sandbox.stub().resolves(queryResult))
+			.stdout()
+			.command(['account:get', addressesWithEmpty.join(',')])
+			.it(
+				'should get accounts info only using non-empty args and display as an array',
+				() => {
+					expect(api.default).to.be.calledWithExactly(apiConfig);
+					expect(query.default).to.be.calledWithExactly(
+						apiClientStub,
+						endpoint,
+						[
+							{
+								query: {
+									limit: 1,
+									address: addresses[0],
```

Comment:
It's the same thing, but would make more sense if these were both `addressesWithEmpty[0]`. Probably also applies to the blocks test.

Reply:
🤕